config/manager/manager.yaml

```diff
@@ -44,7 +21,7 @@ spec:
       containers:
       - command:
         - /manager
-        image: controller:latest
+        image: github.com/kubeflow/kfserving/cmd/manager
```

Comment:
This seems weird to me.

Reply:
this is how `KO` works, in the manifest you put the image name with the import path of the cmd and `KO` knows which cmd you want to build the image for and later image name gets patched in the default and development overlays. I replicated this from knative serving(https://github.com/knative/serving/blob/master/config/controller.yaml#L40).

Reply:
How do ko and kustomize work together? Feels like this should be in a kustomize patch for dev?

Reply:
That’s exactly what I did in the image patch script on this PR, I use ko to resolve the image with a digest built from your local code and then create a kustomize patch for dev.

Reply:
Awesome Dan thanks!